skada/deep/base.py

```diff
@@ -605,7 +589,7 @@ def _get_predict_nonlinearity(self):
             raise TypeError("predict_nonlinearity has to be a callable, 'auto' or None")
         return nonlin
 
-    def predict_proba(
+    def predict_DA_proba(
```

Comment:
why do you want to call it `predict_DA_proba`?
maybe you should remove it, wdyt?